src/components/pages/books/BookItem.js

```diff
@@ -31,7 +52,7 @@ function BookItem(props) {
 BookItem.propTypes = {
   id: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
-
+  category: PropTypes.string.isRequired,
 };
 
```

Comment:
- [ ] There seems to be a bug in your app. When I add a book for the first time, the **category** attribute does not show up on the page. **`Fiction`** is your default category (according to your API setup). As can be seen in the image below, I added the book **Dune** but it has no **`Fiction`** category, and there are console errors. I need to reload the page or go from the `Books` route to the `Categories` route and then back for the category to show up. Kindly fix this.

  **SUGGESTION**: One way this can be fixed is to remove `.isRequired` from category here. And then set a default value for the category prop attribute. So that you have something like this.

  ```
  ...
    category: PropTypes.string,
  };
  
  BookItem.defaultProps = {
    category: 'Fiction',
  };
  ```

  ![abdulai-dune](https://user-images.githubusercontent.com/36542986/150362030-296364e2-a450-4de7-b583-f1c26dcd33cb.jpg)

Reply:
Thank you for the feedback, will work on it